Zadanie2.rb

```diff
@@ -0,0 +1,105 @@
+class Character
+  attr_accessor :name, :level
+	
+  def initialize(name: , level:)
+    self.name= name.class==String ? name: "NoName"
+    self.level= (level.class==Integer && level>0 && level<100)? level:1 
+  end
+
+  def strength
+   level+[*1..12].sample
+  end
+
+  def card
+    puts "#{name} (#{level} lvl)"
+  end
+end
+
+class Monster<Character
+	def initialize(name:,level:)
+		super(name: name,level: level)
+	end
+end
+
+
+class Warrior<Character
+	def initialize(name:,level:)

```

Comment:
te `initialize` niepotrzebne zupełnie, jeśli nie znajdzie w `Warrior` / `Monster` to będzie szukał w `Character` 👀